.gitignore

```diff
@@ -7,6 +7,8 @@
 .env
 .env.test
 
+/vendor
+
```

Comment:
What is this `vendor` directory and why do we add it to .gitignore ?

Reply:
While working, `/vendor` folder was showing +7000 new files and those were mainly only gems for ruby, but this was my fault while I was testing it generated that specific folder. I will remove that in the next commit.